observability/micrometer/src/main/resources/org/wildfly/extension/micrometer/LocalDescriptions.properties

```diff
@@ -10,3 +10,6 @@ micrometer.remove=Operation Removes subsystem
 micrometer.exposed-subsystems=The names of the WildFly subsystems that exposes their metrics (or '*' to expose any subsystem metrics).
 micrometer.endpoint=The URL to which metrics will be published.
 micrometer.step=The step size (reporting frequency) to use in seconds.
+micrometer.registry=Micrometer registries
+micrometer.registry.prometheus.context=The context root from which to serve Prometheus metrics (e.g., "/metrics" or "/prometheus"). Note: Prometheus is not supported on host controllers.
+micrometer.registry.prometheus.security-enabled=Whether or not the context should be secured
```

Comment:
This file seems really incomplete. I don't see how it works; I suspect there's a bug being relied upon!

Reply:
@jasondlee It relies on a bug where if text is missing the description uses higher level text.

There should be these items:

micrometer.registry.otlp
micrometer.registry.otlp.add
micrometer.registry.otlp.remove
micrometer.registry.prometheus
micrometer.registry.prometheus.add
micrometer.registry.prometheus.remove

Plus, in theory, micrometer.registry.xxx.yyyy for each attribute of the respective 'xxx'. You have those for  'prometheus'. It seems to work fine without those though, perhaps due to the alias thing. So don't worry about that bit.

Reply:
Added anyway. :P